multy_core/src/bitcoin/bitcoin_transaction.cpp

```diff
@@ -716,8 +722,32 @@ void BitcoinTransaction::update()
             // not setting value with set_value(), since that would fail for read-only property.
             change_destination->amount.get_value() += remainder;
         }
+
+        sign();
+        counter_stream.reset_byte_count();
```

Comment:
Please move getting TX size info a separate function, which is going to be used as:
```
const uint64_t tx_size = get_transaction_serialized_size(*this, WITH_ZERO_CHANGE);
```